backends/vulkan/test/vulkan_compute_api_test.cpp

```diff
@@ -748,15 +747,14 @@ void run_from_gpu_test(
   vTensor vten =
       vTensor(api::context(), sizes, api::kFloat, storage_type, memory_layout);
 
-  std::stringstream kernel_name;
-  kernel_name << "idx_fill_texture";
-  apply_memory_layout_suffix(kernel_name, vten);
-  apply_dtype_suffix(kernel_name, vten);
+  std::string kernel_name("idx_fill_texture");
+  add_memory_layout_suffix(kernel_name, vten);
+  add_dtype_suffix(kernel_name, vten);
```

Comment:
ditto missing reserve; inline size is as little as 16 bytes with libstdc++ (which I don't know if we use).